packages/amplify-migration-tests/src/migration-helpers/utils.ts

```diff
@@ -1,4 +1,16 @@
 import { v4 as uuid } from 'uuid';
+import {
+  cliInputsExists,
+  getBackendConfig,
+  getCLIInputs,
+  getCloudFormationTemplate,
+  getParameters,
+  parametersExists,
+} from '@aws-amplify/amplify-e2e-core';
+import * as cfnDiff from '@aws-cdk/cloudformation-diff';
```

Comment:
Is the assumption that Cdkv1 and Cdkv2 output is standardized CFN templates - so there is no need to worry that the cfnDiff utility only works when comparing v2 <-> v2 or v1 <-> v1?

I guess what I'm asking is: since this tool is imported from aws-cdk, it works when diffing cdkv1 & cdkv2 CFN templates? Seems like it should, but thats my guess

Reply:
Yes, that's the assumption. CFN template doesn't know which CDK generated it. It has to conform to CFN spec which is not something CDK have control on.